### PR TITLE
LP-2289 Fix all linting errors for philu_courseware

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
### Description

[LP-2289](https://philanthropyu.atlassian.net/browse/LP-2289) Fix all linting errors for philu_courseware

### CODE REVIEW

File: openedx/features/philu_courseware/tests/test_competency_assessment_record_manager.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+"""
+All test cases for competency assessment record manager
+"""
 from django.test import TestCase
 from opaque_keys.edx.keys import UsageKey
```
Comment (MunamMubashir-arbisoft) [inline openedx/features/philu_courseware/tests/test_competency_assessment_record_manager.py:5]: Kindly add a newline between Django imports and project imports.

File: openedx/features/philu_courseware/views.py [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 Views to add features in courseware.
 """
-
 from django.utils.translation import ugettext as _
 from opaque_keys import InvalidKeyError
-from opaque_keys.edx.django.models import CourseKey, UsageKey
+from opaque_keys.edx.django.models import CourseKey
 from rest_framework import status
 from rest_framework.exceptions import APIException, ValidationError
 from rest_framework.response import Response
```
Comment (MunamMubashir-arbisoft) [inline openedx/features/philu_courseware/views.py:9]: Shouldn't we separate opaque key imports from django and DRF imports?

**Review by MunamMubashir-arbisoft — CHANGES_REQUESTED** [review]
Great work!
Just a couple suggestions I want you to take a look at.

File: openedx/features/philu_courseware/tests/test_competency_assessment_record_manager.py (hunk 1 above)
Comment (zahid-arbisoft) [reply openedx/features/philu_courseware/tests/test_competency_assessment_record_manager.py:5]: We run isort and do not manage imports ordering and spacing manually. This is the result of isort.

File: openedx/features/philu_courseware/views.py (hunk 2 above)
Comment (zahid-arbisoft) [reply openedx/features/philu_courseware/views.py:9]: We run isort and do not manage imports ordering and spacing manually. This is the result of isort.

File: openedx/features/philu_courseware/helpers.py [hunk 3]
```diff
@@ -15,6 +18,17 @@
...
+    """
+    Create and return link of a specific chapter in a course
+
+    Args:
+        course (CourseOverview): Course for which we want to send email
+        chapter_index (int): Chapter index for which url be returned
+
+    Returns:
+        str: Link to course chapter
+
```
[the remark this answers is not in the document]
Comment (zahid-arbisoft) [reply openedx/features/philu_courseware/helpers.py:30]: done

**Review by MunamMubashir-arbisoft — APPROVED** [review]
Great work! 💯 
And thanks for clearing out the question regarding imports.

**Review by annas-usman — APPROVED** [review]
I have mentioned a few comments, everything else LGTM

File: openedx/features/philu_courseware/serializers.py [hunk 4]
```diff
@@ -1,10 +1,16 @@
...
 
 from .helpers import validate_problem_id
 from .models import CompetencyAssessmentRecord
 
 
 class CompetencyAssessmentRecordSerializer(serializers.ModelSerializer):
+    """
+    Serializer for model CompetencyAssessmentRecord
+    """
     user = serializers.HiddenField(default=serializers.CurrentUserDefault())
```
[the remark this answers is not in the document]
Comment (zahid-arbisoft) [reply openedx/features/philu_courseware/serializers.py:14]: done

File: openedx/features/philu_courseware/models.py [hunk 5]
```diff
@@ -108,6 +118,15 @@ def __unicode__(self):
 
 
 class CourseEnrollmentMeta(models.Model):
+    """
+    A model to establish linkages between course enrollment and specialisation through which it is being enrolled in
+    """
     course_enrollment = models.OneToOneField(CourseEnrollment, related_name='course_enrollment_meta',
```
[the remark this answers is not in the document]
Comment (zahid-arbisoft) [reply openedx/features/philu_courseware/models.py:124]: done

**Pull request merged**